SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -73,20 +73,22 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 	FString ClientWorkerAttribute = GetOwnerWorkerAttribute(Actor);
 
 	WorkerAttributeSet ServerAttribute = { SpatialConstants::ServerWorkerType };
+	WorkerAttributeSet AIWorkerAttribute = { FString(TEXT("AIWorker")) };
 	WorkerAttributeSet ClientAttribute = { SpatialConstants::ClientWorkerType };
 	WorkerAttributeSet OwningClientAttribute = { ClientWorkerAttribute };
 
 	WorkerRequirementSet ServersOnly = { ServerAttribute };
+	WorkerRequirementSet AllServerTypes = { ServerAttribute, AIWorkerAttribute };
 	WorkerRequirementSet ClientsOnly = { ClientAttribute };
 	WorkerRequirementSet OwningClientOnly = { OwningClientAttribute };
 
-	WorkerRequirementSet AnyUnrealServerOrClient = { ServerAttribute, ClientAttribute };
-	WorkerRequirementSet AnyUnrealServerOrOwningClient = { ServerAttribute, OwningClientAttribute };
+	WorkerRequirementSet AnyUnrealServerOrClient = { ServerAttribute, AIWorkerAttribute, ClientAttribute };
+	WorkerRequirementSet AnyUnrealServerOrOwningClient = { ServerAttribute, AIWorkerAttribute, OwningClientAttribute };
 
 	WorkerRequirementSet ReadAcl;
 	if (Class->HasAnySpatialClassFlags(SPATIALCLASS_ServerOnly))
 	{
-		ReadAcl = ServersOnly;
+		ReadAcl = AllServerTypes;
```

Comment:
This might be an issue, let's chat offline.

Reply:
Is this resolved?

Reply:
Yes, we will keep an eye on the performance impact on this when we run UNR-1079